module_24_Python/1_taxes.py

```diff
@@ -0,0 +1,148 @@
+"""
+Задача 1. Налоги
+Реализуйте иерархию классов, описывающих имущество налогоплательщиков.
+Она должна состоять из базового класса Property
+и производных от него классов Apartment, Car и CountryHouse.
+Базовый класс должен иметь атрибут worth (стоимость), который передаётся в конструктор,
+и метод расчёта налога,
+переопределённый в каждом из производных классов.
+Налог на квартиру вычисляется как 1/1000 её стоимости, на машину — 1/200, на дачу — 1/500.
+Каждый дочерний класс должен иметь конструктор с одним параметром,
+передающий свой параметр конструктору базового класса.
+
+Разработайте интерфейс программы.
+Пусть она запрашивает у пользователя количество его денег и стоимость имущества,
+а затем выдаёт налог на соответствующее имущество и показывает,
+сколько денег ему не хватает (если это так).
+"""
+
+
+class Property:
+    """
+    class Property contains property worth
+
+    Args:
+        worth (float): worth of property
+        Attributes:
+            tax_rate (int): tax rate
+
+    """
+
+    tax_rate = 1000
```

Comment:
No default tax_rate should be specified.
Make it `tax_rate: int`

Reply:
And add `_worth: int`

Reply:
And since tax_rate is a ratio, it should be `tax_rate: float`

Reply:
done